tests/service/data/TestTP/TestTP.fsproj

```diff
@@ -2,7 +2,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net45</TargetFramework>
+    <TargetFramework>net472</TargetFramework>
```

Comment:
could this also be netcoreapp?

Reply:
The purpose of this PR, the test is testing the TP on the desktop.  I'm sure the test can be properly parameterized to test a variety of frameworks, but that is not the purpose of this PR.